### PR TITLE
Feature/16 define a style standard

### PR DESCRIPTION
**Pull request by JonasProgrammer**

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (MrPointer) [inline .idea/codeStyles/Project.xml:1]: Is this file truly needed?

**Review by MrPointer — APPROVED** [review]

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (JonasProgrammer) [reply .idea/codeStyles/Project.xml:1]: Well, that's both the files that my beloved sea lion created after opening the project with the latest version.

Looking at https://confluence.jetbrains.com/display/IDEADEV/New+project+code+style+settings+format+in+2017.3 it seems that both files are safe to put in a VCS. While it also states 'the latter [Project.xml] is optional', it then refers it to be created when something differs from the default settings. The 'empty' file being created in my case is probably due to me having some non-standard settings for my default settings, whereas here I rolled back to the default.

Long story short, it seems not to harm anyone and force-pull settings back to default when somebody uses custom settings as their IDE default, thus IMHO it can stay there.

Comment (MrPointer) [reply .idea/codeStyles/Project.xml:1]: Oh interesting, thanks for sharing the update.
